src/main/java/roody/Roody.java

```diff
@@ -0,0 +1,130 @@
+package roody;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Roody {
+    private ArrayList<Task> list;
+    private Storage storage;
+    private Ui ui;
+
+    public Roody(String filepath){
+        // Assumed no more than 100 tasks
+        this.list = new ArrayList<Task>();
+        this.ui = new Ui();
+        this.storage = new Storage(filepath);
+    }
+    
+    // toggles completion status of tasks
+    private void complete(int index, boolean complete) throws RoodyException{
```

Comment:
perhaps a javadoc @param comment here would be helpful to explain why boolean complete is needed